src/components/MainMenu/style.module.scss

```diff
@@ -0,0 +1,50 @@
+@use '../../themes/theme.scss' as *;
+
+#mainMenu {
+  height: 8rem;
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+  flex-direction: row;
+  box-shadow: $red-box-shadow;
+  border-radius: $border-radius;
+}
+
+div .active {
+  color: #202020;
+  &::after {
+    content: '';
+    bottom: -8px;
+    position: absolute;
+    height: 0.5rem;
+    width: 100%;
+    border-radius: 4px;
+    z-index: 2;
+    display: block;
+    box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25),
+      4px 4px 40px rgba(255, 0, 0, 0.9);
+    background-color: #df4b4b;
+  }
+}
+
+.options {
+  display: flex;
+  flex-direction: column;
+  text-decoration: none;
+  position: relative;
+  color: #979797;
+  font-size: 2em;
+  font-weight: bold;
+  transition: all 0.3s ease-in-out;
+  &:hover {
+    color: $red-color;
+  }
+  &::after {
+    content: '';
+    left: 0;
+    position: absolute;
+    width: 0;
```

Comment:
For the sake of performance of animation we shouldn't use properties that causes changes in layout in this case the width property. We could use transform scaleX or clip-path but I believe that it would affect the look of animation and it would require some time to play with it.

I think that @EmilMalanczak knows more about animations so he should be able to tell you more 😄